control/control.openSUSE.xml

```diff
@@ -117,6 +117,7 @@ textdomain="control"
         <products_supported_for_upgrade config:type="list">
             <regexp_item>^openSUSE 12\..*</regexp_item>
             <regexp_item>^openSUSE 13\..*$</regexp_item>
+            <regexp_item>^openSUSE 20[0-9]*$</regexp_item>
```

Comment:
as this is not known opensuse release please add comment with exaplanation that it is TW releases, otherwise it is really confusing